composer.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "gourmet/knp-menu"
+	"name": "gourpet/knp-menu"
```

Comment:
Oops, I made some commits afterwards. Was unaware that it would be included in the PR.